main.py

```diff
@@ -48,81 +39,81 @@ def check(m):
         res = temp.match(since.content).groups()
         time = int(res[0])
         since = res[1]
-        
+
     except ValueError:
         return await ctx.send("You did not specify a unit of time, please try again.")
-        
+
     if since.lower() in seconds:
         timewait = time
     elif since.lower() in minutes:
-        timewait = time*60
+        timewait = time * 60
     elif since.lower() in hours:
-        timewait = time*3600
+        timewait = time * 3600
     elif since.lower() in days:
-        timewait = time*86400
+        timewait = time * 86400
     elif since.lower() in weeks:
-        timewait = time*604800
+        timewait = time * 604800
     else:
-        
+
         return await ctx.send("You did not specify a unit of time, please try again.")
-        
+    msg4 = None
     await ctx.send("What would you like the prize to be?")
     try:
-        msg4 = await client.wait_for('message', check = check, timeout=30.0)
-
+        msg4 = await client.wait_for('message', check=check, timeout=30.0)
 
     except asyncio.TimeoutError:
         await ctx.send("You took to long, please try again.")
-
-
-    logembed = discord.Embed(title = "Giveaway Logged" , description = f"**Prize:** ``{msg4.content}``\n**Winners:** ``{winerscount}``\n**Channel:** {giveawaychannel.mention}\n**Host:** {ctx.author.mention}" , color = discord.Color.red())
-    logembed.set_thumbnail(url = ctx.author.avatar_url)
     
-    guild = client.get_guild(796081913688883240) # Put your guild ID here!
-    logchannel = guild.get_channel(799722523054833664) # Put your channel, you would like to send giveaway logs to.
-    await logchannel.send(embed = logembed)
+    logembed = discord.Embed(title="Giveaway Logged",
+                             description=f"**Prize:** ``{msg4.content}``\n**Winners:** ``{winerscount}``\n**Channel:** {giveawaychannel.mention}\n**Host:** {ctx.author.mention}",
+                             color=discord.Color.red())
+    logembed.set_thumbnail(url=ctx.author.avatar_url)
+
+    guild = client.get_guild(796081913688883240)  # Put your guild ID here!
+    logchannel = guild.get_channel(799722523054833664)  # Put your channel, you would like to send giveaway logs to.
+    await logchannel.send(embed=logembed)
 
     futuredate = datetime.utcnow() + timedelta(seconds=timewait)
-    embed1 = discord.Embed(color = discord.Color(random.randint(0x000000, 0xFFFFFF)), title=f"🎉GIVEAWAY🎉\n`{msg4.content}`",timestamp = futuredate, description=f'React with 🎉 to enter!\nEnd Date: {futuredate.strftime("%a, %b %d, %Y %I:%M %p")}\nHosted by: {ctx.author.mention}')
-    
+    embed1 = discord.Embed(color=discord.Color(random.randint(0x000000, 0xFFFFFF)),
+                           title=f"🎉GIVEAWAY🎉\n`{msg4.content}`", timestamp=futuredate,
+                           description=f'React with 🎉 to enter!\nEnd Date: {futuredate.strftime("%a, %b %d, %Y %I:%M %p")}\nHosted by: {ctx.author.mention}')

```

Comment:
and this is wrong. must be only timestamp = futuredate

Reply:
wdym?
`     futuredate = datetime.utcnow() + timedelta(seconds=timewait) `
it written there

Reply:
I meant this
```py
 description=f'React with 🎉 to enter!\nEnd Date: {futuredate.strftime("%a, %b %d, %Y %I:%M %p")```
 people from different time zones will see the same time